openshift-client/src/main/java/io/fabric8/openshift/client/DefaultOpenShiftClient.java

```diff
@@ -140,6 +140,10 @@ public DefaultOpenShiftClient() throws KubernetesClientException {
     this(new OpenShiftConfigBuilder().build());
   }
 
+  public DefaultOpenShiftClient(boolean disableApiGroupCheck) throws KubernetesClientException {
```

Comment:
Personally, I wouldn't add an new constructor just for this. That's what the Config objects is here for. For things that are TOO common and TOO obvious (e.g. the master url) we can make an exception, but I don't think that this is the case here.